lib/job.js

```diff
@@ -3,12 +3,16 @@
 const logger = require('screwdriver-logger');
 const dayjs = require('dayjs');
 const hoek = require('@hapi/hoek');
+const { PR_JOB_NAME } = require('screwdriver-data-schema').config.regex;
 const BaseModel = require('./base');
 const { getAnnotations, getAllRecords, getToken } = require('./helper');
 const executor = Symbol('executor');
 const tokenGen = Symbol('tokenGen');
 const apiUri = Symbol('apiUri');
-const START_INDEX = 3;
+const REGEX_CAPTURING_GROUP = {
+    pr: 1, // PR-1
+    job: 2 // main or undefined if using legacy
```

Comment:
We need to update the comment. `null` or `undefined` is overwritten to "main" in `pipeline` model

Reply:
?

Reply:
The logic to return "main" when job name is `undefined` is in [`pipeline` model](https://github.com/screwdriver-cd/models/pull/537/files#diff-2977a16e13f278992ce964024e936111d8f1c3b68f5f698c7e06195258912721R455). Having this comment in `job` model could be confusing as `job` model does not do any overrides by itself.